src/Features/Core/Portable/SplitOrMergeIfStatements/IIfLikeStatementGenerator.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.SplitOrMergeIfStatements
+{
+    /// <summary>
+    /// An if-like statement is either an if statement or an else-if clause.
+    /// When querying the syntax, C# else if chains are "flattened" and modeled to look like VB else-if clauses,
+    /// so an if-like statement can be followed a sequence of else-if clauses (which are themselves if-like statements)
+    /// and an optional final else clause. These else-if clauses are treated as independent when removing or inserting.
+    /// </summary>
+    internal interface IIfLikeStatementGenerator : ILanguageService
+    {
+        bool IsIfLikeStatement(SyntaxNode node);
```

Comment:
Is just IsIfOrElseIf clearer? Then I don't have to learn newfangled terminology.

Reply:
➡️ Fixed in b1e9d156e7bc1db889fba279a8df8499600d4cea